readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Grifion.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Grifion.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="952" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/b41ed9b9-7c6e-442e-8b7d-9adb4249d2f9">